sound/soc/intel/skylake/skl-sst-utils.c

```diff
@@ -311,6 +311,9 @@ int snd_skl_parse_uuids(struct sst_dsp *ctx, const struct firmware *fw,
 
 	return 0;
 
+free_uuid_module:
+	kfree(module);
+
```

Comment:
@aiChaoSONG please drop this patch, we don't use/own this driver.

@crojewsk @crojewsk-intel please look into this on your side.

Reply:
@plbossart sure thing, will do. Sorry for the delay, came back from holidays recently and didn't notice the request at first.